<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="delayCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>355</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Задержка</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="delaySpinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>355</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>390</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет заполнения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>480</y>
      <width>265</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить всё</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>520</y>
      <width>265</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fillButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>450</y>
      <width>265</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнить область</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>420</y>
      <width>265</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Замкнуть кривую</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>280</width>
      <height>271</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>285</y>
      <width>10</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="xspinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>285</y>
      <width>10</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="yspinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>315</y>
      <width>265</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить точку</string>
    </property>
   </widget>
   <widget class="DrawLabel" name="drawLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>800</width>
      <height>720</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawLabel</class>
   <extends>QLabel</extends>
   <header location="global">drawlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
